03/index.html

```diff
@@ -1,16 +1,48 @@
 <!DOCTYPE html>
 <html lang="pl">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="/03/styles/global.css">
+    <link rel="stylesheet" href="/03/styles/mobile.css">
+    <link rel="stylesheet" media="(min-width: 600px)" href="/03/styles/tablet.css">
+    <link rel="stylesheet" media="(min-width: 1200px)" href="/03/styles/desktop.css">
+
     <title>devmentor.pl - HTML & CSS: RWD - #03</title>
 </head>
+
 <body>
-    
-    <main class="t">T</main>
-    <header class="p">P</header>
-    <side class="v">V</side>
+
+    <main class="t">
+        <p>
+            Lorem ipsum, dolor sit amet consectetur adipisicing elit. Architecto est autem debitis harum perferendis,
+            mollitia doloribus, numquam id consequatur voluptatum tempora explicabo, laboriosam omnis? Molestias fugit
+            laboriosam, suscipit ab quo amet accusamus. Soluta, eos voluptas ipsa distinctio aliquam ea esse eum
+            repudiandae quaerat debitis cumque quam, iusto fuga qui nostrum velit similique tempore quas commodi!
+            Provident ut quibusdam iste assumenda, voluptatibus, quisquam a eos accusamus magni molestiae similique
+            vitae in reprehenderit illo explicabo enim non saepe obcaecati eius! Esse cumque recusandae magnam
+            necessitatibus voluptate tempore suscipit, nulla facilis voluptatum nemo, quae, pariatur facere temporibus.
+            Veritatis modi laudantium deleniti explicabo voluptas. Ipsa amet, illum commodi neque ad quis harum
+            perspiciatis provident delectus, nulla repellat facere debitis fugiat error aut voluptate minima.
+        </p>
+    </main>
+    <header class="p">
+        <picture>
+            <source srcset="./assets/img/halloween-700px.png" media="(min-width: 1199px)">
+            <source srcset="./assets/img/halloween-500px.png" media="(min-width: 600px)">
+
+            <img src="./assets/img/halloween-600px.png" alt="Hallowen">
```

Comment:
Obrazkowi należy nadać `max-width: 100%`, aby się dostosowywał do wielkości okna (jeśli jest większy niż ono):

![image](https://github.com/devmentor-pl/practice-html-and-css-rwd/assets/55095257/3d46ccd5-be0a-4056-84c0-8a971067a120)

PS. Często w pliku reset.css jest juz to zadklarowane ;)